Gemfile

```diff
@@ -49,6 +49,9 @@ gem "aws-sdk-s3"
 gem "paper_trail"
 # Store active record objects in version whodunnits
 gem "paper_trail-globalid"
+# Receive exceptions and configure alerts
+gem "sentry-rails", "~> 5.0", ">= 5.0.2"
```

Comment:
Do we need the min and the twiddle waka?

Reply:
Gone